lib/base/error-handler.js

```diff
@@ -87,17 +87,19 @@ const log = require('npmlog'),
             "EMPTY_VALUE" : "Please specify a value",
             "EXISTING_VALUE" : "The specified value already exists",
             "EXISTING_FILETYPE_PATH" : "A file with the same name at the specified location already exists",
+            "NOT_MATCHED_LOGDAEMON" : "CLI's log daemon should matched with the target's log deamon.\nPlease change CLI's log daemon to same as target's log daemon",
```

Comment:
`CLI's log deamon should match with the target's log deamon (without ".")  `

Maybe we need to SW Doc help

Reply:
done